blocks/header/header.js

```diff
@@ -103,7 +113,7 @@ export default async function decorate(block) {
     nav.id = 'nav';
     nav.innerHTML = html;
 
-    const classes = ['brand', 'sections', 'tools'];
+    const classes = ['meteo', 'brand', 'sections', 'tools'];
```

Comment:
instead of class tools use language?

Reply:
good catch, we don't even have that grid area anymore.